dair_pll/multibody_learnable_system.py

```diff
@@ -388,41 +326,22 @@ def sim_step(self, x: Tensor, carry: Tensor) -> Tuple[Tensor, Tensor]:
         v_plus = self.forward_dynamics(q, v, u)
         return v_plus, carry
 
-    def summary(self, statistics: Dict, new_geometry: bool = False
-    ) -> SystemSummary:
+    def summary(self, statistics: Dict) -> SystemSummary:
         """Generates summary statistics for multibody system.
 
         The scalars returned are simply the scalar description of the
-        system's :py:class:`MultibodyTerms`. Additionally, two trajectory pairs
-        are visualized as videos, one for training set and one for validation
-        set.
+        system's :py:class:`MultibodyTerms`.
+
+        Meshes are generated for learned
+        :py:class:`~dair_pll.geometry.DeepSupportConvex` es.
 
         Args:
             statistics: Updated evaluation statistics for the model.
-            new_geometry: Whether to use new learned geometry or the original
-              URDF's geometry in the rollout videos.
 
         Returns:
-            Scalars and videos packaged into a ``SystemSummary``.
+            Scalars and meshes packaged into a ``SystemSummary``.
```

Comment:
Before these changes, this `summary()` method actually did create a complete `SystemSummary`, full of meshes, scalars, and videos.  Now, this function does scalars and meshes, while `DrakeExperiment.joint_system_summary()` does videos, so my guess is that for a complete summary both of these methods need to be called and the results combined.  It could be helpful to add some comments that speak to how this incomplete `summary()` relates to other methods that also do partial summaries.  Maybe different naming choices could make this more straightforward.

Reply:
See previous comments on learned system summary vs. base-vs-learned system comparison summary.